src/layouts/shared/page-header.tsx

```diff
@@ -49,6 +51,15 @@ export default function PageHeader() {
               )}
             </Link>
           </h1>
+          {hasContactFormUrl && (
```

Comment:
Let us add a `nav` element around this link element 👍 Also, should this be moved out of the element with the `pageHeader__inner` class? The link looks a bit off when it's placed all the way to the right 🤔

Reply:
Do you think it looks better when the link is placed to closer to the logo?

Reply:
I think it should look more like this: 

![image](https://github.com/user-attachments/assets/06ab8710-2831-4030-812e-e82b5dbb6c07)

![image](https://github.com/user-attachments/assets/54c4c851-0e6b-46cb-aa17-2d6733f5f81d)

Reply:
Those screenshots are from the webshop, this is from the current implementation in the planner-web:

![image](https://github.com/user-attachments/assets/7b587cba-d244-485d-8a34-6b8b6d02bf57)

Reply:
![Skjermbilde 2024-08-26 kl  12 39 44](https://github.com/user-attachments/assets/b08f8ab0-b6cf-486d-894d-1d076c48b8ca)